docs/website/content/v0.4/en/guides/getting-started/firecracker.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Firecracker
+---
+
+In this guide we will create a Kubernetes cluster using Firecracker.
+
+## Requirements
+
+- at least firecracker v
+- CNI (`CAP_SYS_ADMIN` and `CAP_NET_ADMIN` capabilities are required)
+- iptables
+- KVM
+
+## Create the Cluster
+
+```bash
+talosctl cluster create --wait --provisioner firecracker
+```
+
+Once the above finishes successfully, your talosconfig(`~/.talos/config`) will be configured to point to the new cluster.
+
+> Note: You must run the above as `root`.
```

Comment:
we may wanna add the actual sudo command here. I think it's something like `sudo -E talosctl cluster create ...`